duckduckgo/macos-browser
# Remove navigatorCredentials

Task/Issue URL:  https://app.asana.com/0/1163321984198618/1201071474741984/f and https://app.asana.com/0/1200437802575119/1201288740667760/f
Tech Design URL:
CC:

**Description**:

Removes navigatorCredentials.js and moves it to: https://github.com/duckduckgo/content-scope-scripts/pull/5

- Relies on https://github.com/duckduckgo/BrowserServicesKit/pull/47

This is a follow up from: https://github.com/more-duckduckgo-org/macos-browser/pull/389 **(merged)**

**Steps to test this PR**:
1. Open imgur.com and see if it loads
1. In the web inspector ensure navigator.credentials.get() exists

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
